packaging/windows/doinstaller_wix.cmd

```diff
@@ -127,7 +127,19 @@ IF %ERRORLEVEL% NEQ 0 (
   EXIT /B 1
 )
 
-if NOT DEFINED JSON_UPDATE_FILENAME set JSON_UPDATE_FILENAME=update-windows-prod-v2.json
+:: UpdateChannel is a Jenkins select parameter, one of: Smoke, Test, None
+echo UpdateChannel: %UpdateChannel%
+set JSON_UPDATE_FILENAME=update-windows-prod-v2.json
+IF %UpdateChannel% EQU Test (
+  set JSON_UPDATE_FILENAME=update-windows-prod-test-v2.json
+)
+IF %UpdateChannel% EQU Smoke (
+  set JSON_UPDATE_FILENAME=update-windows-prod-%KEYBASE_VERSION%.json
+)
+IF %UpdateChannel% EQU Smoke2 (
+  set JSON_UPDATE_FILENAME=update-windows-prod-%KEYBASE_VERSION%.json
+)
```

Comment:
Would it make sense to detect unexpected values of `UpdateChannel` here and freak out?